src/components/enterprise-user-subsidy/enterprise-offers/EnterpriseOffersBalanceAlert.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useContext } from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
+import { Container, Alert, MailtoLink } from '@edx/paragon';
+import { AppContext } from '@edx/frontend-platform/react';
+
+const EnterpriseOffersBalanceAlert = ({
+  adminText,
+  alertClassName,
+  alertVariant,
+  alertIcon,
+  alertHeading,
+  alertText,
+}) => {
+  const {
+    enterpriseConfig: { adminUsers },
+  } = useContext(AppContext);
+
+  const adminEmails = adminUsers.map(user => user.email);
+  const hasAdminEmails = adminEmails.length > 0;
+
+  const actions = hasAdminEmails ? [
+    <MailtoLink
```

Comment:
Do we want to instrument this button with a Segment event so we can know if/when learners click it?